src/main/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImpl.scala

```diff
@@ -121,20 +122,10 @@ private[launcher] class OfferProcessorImpl(
   private[this] def saveTasks(ops: Seq[TaskOpWithSource], savingDeadline: Timestamp): Future[Seq[TaskOpWithSource]] = {
     def saveTask(taskOpWithSource: TaskOpWithSource): Future[Option[TaskOpWithSource]] = {
       val taskId = taskOpWithSource.taskId
-
-      val persistedOp = taskOpWithSource.op.maybeNewTask match {
-        case Some(newTask) =>
-          log.info(
-            s"Save ${taskOpWithSource.taskId} " +
-              s"after applying the effects of ${taskOpWithSource.op.getClass.getSimpleName}"
-          )
-          taskCreationHandler.created(newTask)
-        case None =>
-          log.info(s"Remove ${taskOpWithSource.taskId} because of ${taskOpWithSource.op.getClass.getSimpleName}")
-          taskCreationHandler.terminated(taskId)
-      }
-
-      persistedOp.map(_ => Some(taskOpWithSource))
+      log.info("Save task [{}]", taskOpWithSource.taskId)
```

Comment:
you can already change the logging "Persisting TaskStateOp"

In fact, you can already rename the `newTask` field -- nobody is working on this concurrently. At least it confused me...

Have you documented that you want to change taskCreationHandler.created somewhere? Do you keep track of planned renames somewhere? We have this issue for that: https://github.com/mesosphere/marathon/issues/3311